src/js/model/storage.js

```diff
@@ -19,7 +19,8 @@ Object.assign(Storage.prototype, {
         return this.items.reduce((memo, key) => {
             const val = storage[`${this.namespace}.${key}`];
             if (val) {
-                memo[key] = serialize(val);
+                // eslint-disable-next-line no-unused-expressions
```

Comment:
you can abide by the lint rule by changing this to
```
memo[key] = key !== 'captions' ? serialize(val) : JSON.parse(val);
```

Reply:
Adding it!